react-native/react/engine/server.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
```

Comment:
Can we put in some comments about what this is and how it works, very hard to tell